tests/scripts/thread-cert/common.py

```diff
@@ -37,6 +37,8 @@
 # Map of 2 bits of parent priority.
 pp_map = {1: 1, 0: 0, 3: -1, 2: -2}
 
+UDP_TEST_PORT = 12345
```

Comment:
move to config.py ?

Reply:
config.py is imported by ipv6.py, causing circular imports.